templates/applicantForm.phtml

```diff
@@ -18,50 +18,50 @@
         <form>
             <label for="name">Applicant Name</label>
             <div id="nameError" class="alert hidden formItem_alert"></div>
-            <input id="name" name="name" class="submitApplicant form-control" type="text" placeholder="name" required>
+            <input id="name" name="name" class="submitApplicant form-control" type="text" placeholder="name" value="<?php echo $applicantName ?>" required>
```

Comment:
missing `;` on all these echos. also why doesnt this throw loads of warnings on the register page when the variables dont exist? this makes me feel uneasy